### PR TITLE
fix(DB/Creature) Wrathscale Sorceress casts Frostbolt

### PR DESCRIPTION
**Pull request by Vanheden**
## Changes Proposed:
Creature casts Frostbolt.

## Issues Addressed:
Wrathscale Sorceress casts Frostbolt randomly on target.
[Video](https://www.youtube.com/watch?v=aNoKPrcZIwc&ab_channel=Roddan)
Closes: https://github.com/azerothcore/azerothcore-wotlk/issues/16899

## SOURCE:
Retail & Youtube & Sniff.

## Tests Performed:
Teleport to Bloodmyst Isle and attack it.

## How to Test the Changes:
1. Teleport to Bloodmyst Isle or Spawn the NPC.

## Known Issues and TODO List:
None so far.

### CODE REVIEW

File: data/sql/updates/pending_db_world/WrathscaleSorceressFix.sql [hunk 1]
```diff
@@ -0,0 +1,10 @@
+ -- Wrathscale Sorceress smart ai
+SET @ENTRY := 17336;
+DELETE FROM `smart_scripts` WHERE `source_type` = 0 AND `entryOrGuid` = @ENTRY;
+UPDATE `creature_template` SET `AIName` = 'SmartAI', `ScriptName` = '' WHERE `entry` = @ENTRY;
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES 
+(@ENTRY, 0, 0, 0, 0, 0, 100, 0, 3500, 12000, 3500, 12000, 11, 9672, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 'Every 3.5 - 12 seconds (IC) - Self: Cast spell Frostbolt (9672) on Victim');
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/WrathscaleSorceressFix.sql:6]: Same as the other PR comment wise.

name - event - action

`Wrathscale Sorceress - In Combat - Cast 'Frostbolt'`

File: data/sql/updates/pending_db_world/WrathscaleSorceressFix.sql [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+
+DELETE FROM `conditions` WHERE `SourceTypeOrReferenceId` = 22 AND `SourceEntry` = 17336 AND `SourceId` = 0;
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/WrathscaleSorceressFix.sql:9]: Why are we deleting stuff that doesnt exist in the db?

Comment (Vanheden) [reply data/sql/updates/pending_db_world/WrathscaleSorceressFix.sql:9]: I think it's the program I'm using that does it automatically.

Comment (Kitzunu) [reply data/sql/updates/pending_db_world/WrathscaleSorceressFix.sql:9]: delete it

**Comment by GastonCK** [comment]
Author provided a video testing it, does this need another test?

**Review by Kitzunu — APPROVED** [review]

**Comment by Kitzunu** [comment]
Looks good to me. Thanks  =)

**Pull request merged**